Doc/reference/datamodel.rst

```diff
@@ -2092,13 +2092,18 @@ Resolving MRO entries
    :class:`type`, then an ``__mro_entries__`` method is searched on the base.
    If an ``__mro_entries__`` method is found, the base is substituted with the
    result of a call to ``__mro_entries__`` when creating the class.
```

Comment:
```suggestion
   :class:`type`, then an :meth:`!__mro_entries__` method is searched on the base.
   If an :meth:`!__mro_entries__` method is found, the base is substituted with the
   result of a call to :meth:`!__mro_entries__` when creating the class.
```

Format these as (unresolved) methods rather than generic code snippit literals (which are rendered differently, and have different semantic meaning).